instagram-app/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import dummyData from './dummy-data';
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from "./components/PostContainer/PostContainer";
+
+import './App.css';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      postData: dummyData
+    };
+  }
+
+  render () {
+    return (
+      <div className="App">
+        <SearchBar />
+        <PostContainer 
+        postData = {this.state.postData}
```

Comment:
Properly indent code. Sounds nitpicky but clean code is important and makes reading easier.
```
        <PostContainer 
          postData = {this.state.postData}
        />
```